Jojoucsd/day2.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+	<link rel="stylesheet" href="style.css">
+
+	<title>Day2HW Bootstrap</title>
+</head>	
+<body>
+<nav class="navbar navbar-inverse">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <a class="navbar-brand" href="#">Goals</a>
+    </div>
+
+    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+      <ul class="nav navbar-nav navbar-right">
+        <li><a href="#">Button</a></li>
+        <li class="dropdown">
+          <a class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Dropdown <span class="caret"></span></a>
+          <ul class="dropdown-menu">
+            <li><a href="#">Action</a></li>
+            <li><a href="#">Another action</a></li>
+            <li><a href="#">Something else here</a></li>
+            <li role="separator" class="divider"></li>
+            <li><a href="#">Separated link</a></li>
+          </ul>
+        </li>
+      </ul>
+    </div>
+  </div>
+</nav>
+<div>
+  <div class="container1">
+	<div class="row">
+	  <div class="col-sm-offset-1, col-sm-6">
+		<ol>
+		<li>My Day 1 HW turn in was earse due to my mistake on the git push attemp</li>
+		<li>Tring to make up for yesterday, I am including some of the html elements from day 1 in this day 2 hw turn in</li>
+		<li>The most understanding I learn from Day 1 is believe in to myself and embrace the tasks. Accept learning is a process that takes pratice and time.</li>
+	    </ol>
+	  </div>
+	  <div class="col-sm-offset-1, col-sm-6">
+		<ul>The biggest question I have for today's lesson and bootstrap is why the drop down doesn't work. My guess is part of the reason is my jquery isn't working properly in to the the html.
+		</ul>
+		<ul>My concerns are the more libraries I learn the harder for me to debug my mistakes</ul>
+		<ul>I have been struggle deeply in the raw CSS regarding to the building blocks, one of my goal is to review CSS in the weekend</ul>
```

Comment:
CSS isn't something you can really master in a day or even a month.  Just knowing the basics and getting to understand how to make use of it at this point is good.